### ISSUE TITLE
event target: LambdaFunction should support Input property

### ISSUE THREAD

**Issue by cynicaljoy** [awslabs/aws-cdk#2567]
Currently having to use a property override to add my "Input" for my triggered Lambda. The construct should allow for the `Input` to be defined, optionally.

**Comment by cynicaljoy** [comment]
Here's the snippet for anybody who wants to use the property override: 

```typescript
(eventRule.node.findChild('Resource') as cdk.CfnResource).addPropertyOverride('Targets.0.Input', '{"hello": "world"});
```

**Comment by cynicaljoy** [comment]
Available in `0.33.0`

https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-events.RuleTargetInput.html
